UI/src/app/components/panel/panel-target.component.ts

```diff
@@ -0,0 +1,129 @@
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { ITable } from 'src/app/models/table';
+import { BridgeService } from 'src/app/services/bridge.service';
+
+import { CommonService } from 'src/app/services/common.service';
+import { StoreService } from 'src/app/services/store.service';
+import { BridgeButtonService } from '../bridge-button/service/bridge-button.service';
+import { PanelTableComponent } from './panel-table/panel-table.component';
+import { PanelBaseComponent } from './panel-base.component';
+
+const groupsConf = {
```

Comment:
could you create separate json in this directory with this conf info?

Reply:
done